bin/mev/src/cmd/relay.rs

```diff
@@ -1,6 +1,7 @@
 use crate::cmd::config::Config;
 use anyhow::{anyhow, Result};
 use clap::{Args, Subcommand};
+use ethereum_consensus::crypto::SecretKey;
```

Comment:
its preferable to keep this layer of the stack underneath the interface this layer exposes

I left a comment below about how to handle the secret key here which should let us drop these changes